sapirbitan/assignment2/ex1.cpp

```diff
@@ -0,0 +1,126 @@
+#include <iostream>
+using namespace std;
+
+struct node {
+	int key;
+	node* left_child = NULL;
```

Comment:
Use of NULL in C++ is discouraged. You can read why [here](https://stackoverflow.com/questions/20509734/null-vs-nullptr-why-was-it-replaced). Please use nullptr instead here and everywhere else :)